frontend/src/metabase/visualizations/lib/save-dashboard-pdf.ts

```diff
@@ -17,6 +14,7 @@ export const saveDashboardPdf = async (
     return;
   }
 
+  const { default: html2canvas } = await import("html2canvas");
```

Comment:
nit: I'd keep import on the top level, but with `import()`, so the new chunk will be created automatically and at the same time user will not need to wait for chunk to be downloaded after a click to "download"